common/configuration.py

```diff
@@ -76,3 +76,4 @@ def default(cls) -> 'LogRenderer':
 ELASTIC_INDEX = config('ELASTIC_INDEX', default=None)
 ELASTIC_RESULTS_PER_PAGE = config('ELASTIC_RESULTS_PER_PAGE', default=10)
 ELASTIC_SEARCH_TIMEOUT = config('ELASTIC_SEARCH_TIMEOUT', default=25)
+ELASTIC_TOKEN_BALANCES_INDEX = config('ELASTIC_TOKEN_BALANCES_INDEX', default=None)
```

Comment:
You need to add this env variable on the `serverless.yml` and `deploy.yml` files

Reply:
Thanks!

Added in 56b217f9118402cd0360e22f162795303b115324